apigw/src/enduser/routes/auth-weak-login.ts

```diff
@@ -0,0 +1,61 @@
+// SPDX-FileCopyrightText: 2017-2024 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import { z } from 'zod'
+
+import { EvakaSessionUser, login } from '../../shared/auth/index.js'
+import { toRequestHandler } from '../../shared/express.js'
+import { logAuditEvent } from '../../shared/logging.js'
+import { RedisClient } from '../../shared/redis-client.js'
+import { citizenWeakLogin } from '../../shared/service-client.js'
+
+const Request = z.object({
+  username: z.string().min(1).max(128),
```

Comment:
Olisiko tämä sopiva paikka tehdä lowercasetus?
```suggestion
  username: z.string().min(1).max(128).transform((email) => email.toLowerCase()),
```